manifests/app-proxy/kustomization.yaml

```
@@ -3,7 +3,7 @@ kind: Kustomization
 images:
   - name: quay.io/codefresh/cap-app-proxy
     newName: quay.io/codefresh/cap-app-proxy
-    newTag: 1.982.0
+    newTag: 1.983.0
```

```suggestion
    newTag: 1.1004.1
```

it should match the argo-platform version from the pr that fixed the issue (https://github.com/codefresh-io/argo-platform/pull/2039/files#diff-7b032461847f02192bb99ad5d10d6142580e260a8d9a036bda55d1537c57d4daR2)